Fix #14621 : "Propose table structure" on MySQL 8.0

pulunomoe opened the pull request:
Signed-off-by: Agus Perwira Purnomo <furunomail@gmail.com>

### Description

Propose table structure link not displayed anymore for MySQL >= 8.0

Fixes #14621

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

MauricioFauth reviewed (changes requested):
Hello, @furuno. Thanks for your contribution!

Could you use [`QA_4_8`](https://github.com/phpmyadmin/phpmyadmin/tree/QA_4_8) as base branch instead of `master`?

pulunomoe committed:
Fix #14621 - "Propose table structure" on MySQL 8.0

Propose table structure link not displayed anymore for MySQL >= 8.0

Cherry-picked-from: ff6ffb1c35491b18e4096472f0419be934ee65c0
Signed-off-by: Agus Perwira Purnomo <furunomail@gmail.com>

williamdes reviewed (approved):
@furuno Thanks for your contribution !
I rebased the PR onto latest `QA_4_8`
